renovate.json

```diff
@@ -2,5 +2,12 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "local>Ocramius/.github:renovate-config"
-  ]
+  ],
+  "nix": {
+    "enabled": true
+  },
+  "lockFileMaintenance": {
+    "enabled": true,
+    "automerge": true
+  }
```

Comment:
Luckily, this was already enabled in [`local>Ocramius/.github:renovate-config`](https://github.com/Ocramius/.github/blob/c77629415fd5a57ef23d9372075f406e7bf98403/renovate-config.json#L19-L24) :D

Reply:
Removed.